includes/Modules/AdSense.php

```diff
@@ -857,13 +857,11 @@ protected function setup_settings() {
 	 * @return Asset[] List of Asset objects.
 	 */
 	protected function setup_assets() {
-		$base_url = $this->context->url( 'dist/assets/' );
-
 		return array(
 			new Script(
 				'googlesitekit-modules-adsense',
 				array(
-					'src'          => $base_url . 'js/googlesitekit-modules-adsense.js',
+					'version'      => null,
```

Comment:
See other comment, let's keep `src` in here and omit the `version` argument here. This can all be handled internally in the `Script` class.

Reply:
Updated.